desktop/src/main/java/pmstation/integration/Mqtt.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * pm-home-station
+ * 2017-2024 (C) Copyright - https://github.com/rjaros87/pm-home-station
+ * License: GPL 3.0
+ */
+package pmstation.integration;
+
+import com.google.gson.Gson;
+import org.apache.commons.lang3.StringUtils;
+import org.eclipse.paho.client.mqttv3.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pmstation.configuration.Config;
+import pmstation.core.plantower.ParticulateMatterSample;
+
+public class Mqtt {
+    private static final Logger logger = LoggerFactory.getLogger(Mqtt.class);
+    private static final Gson gson = new Gson();
+    private final String topic;
+    private MqttClient client;
+    public static final Mqtt instance = new Mqtt();
+
+    public static Mqtt getInstance() {
+        return instance;
+    }
+
+    private Mqtt() {
+        topic = Config.instance().to().getString(Config.Entry.MQTT_TOPIC.key(),
+                "home/aqi");
+        String broker = Config.instance().to().getString(Config.Entry.MQTT_BROKER.key(),
+                "tcp://localhost:1883");
+        String clientId = Config.instance().to().getString(Config.Entry.MQTT_CLIENT_ID.key(),
+                "PMStationClient");
+        String username = Config.instance().to().getString(Config.Entry.MQTT_USERNAME.key());
+        String password = Config.instance().to().getString(Config.Entry.MQTT_PASSWORD.key());
+
+        MqttConnectOptions options = new MqttConnectOptions();
+        options.setCleanSession(true);
+        if (!StringUtils.isEmpty(username) && !StringUtils.isEmpty(password)) {
+            options.setUserName(username);
+            options.setPassword(password.toCharArray());
+        }
+
+        try {
+            client = new MqttClient(broker, clientId);
+            client.setCallback(new MqttCallback() {
+                @Override
+                public void connectionLost(Throwable cause) {
+                    logger.info("Connection lost!");
+                }
+
+                @Override
+                public void messageArrived(String topic, MqttMessage message) {
+                    logger.info("Message received. Topic: {}, Message: {}", topic, new String(message.getPayload()));
+                }
+
+                @Override
+                public void deliveryComplete(IMqttDeliveryToken token) {}
+            });
+            client.connect(options);
+            client.subscribe(topic);
+        } catch (MqttException e) {
+            logger.error("Unable to create a Mqtt Client", e);
+        }
+    }
+
+    public void publish(ParticulateMatterSample sample) {
+        if (client != null) {
+            MqttMessage message = new MqttMessage();
+            String jsonString = gson.toJson(sample);
```

Comment:
re-format to sth like this:
```
            double temperatureValue = 23.5;
            String temperatureUnit = "Celsius";
            // Construct the payload as a string
            String payload = String.format("{\"value\": %s, \"unit\": \"%s\"}", temperatureValue, temperatureUnit);

            MqttMessage message = new MqttMessage(payload.getBytes());
            client.publish(topic, message);
``` 
(example from chatGPT)

Reply:
units will be implemented in another PR